test/examples/utils/plotting_helpers.jl

```diff
@@ -82,9 +82,14 @@ end
   `p` - additional ODE parameters carried in the solver
 Plots the spectra
 """
-function plot_spectra!(sol, p; file_name = "test_spectra.png", logxrange = (0, 8))
-    x = 10 .^ (collect(range(logxrange[1], logxrange[2], 100)))
-    r = (x * 3 / 4 / π) .^ (1 / 3)
+function plot_spectra!(sol, p; file_name = "test_spectra.png", logxrange = (-2, 9), print = false)
+    x = 10 .^ (collect(range(logxrange[1], logxrange[2], 10)))
```

Comment:
10 points for plotting the spectra is too small. Why do you change 100 to 10?

Reply:
Ah, this was a leftover artifact from my debugging. I'll revert it back to 100.